pywbemtools/pywbemcli/_cmd_listener.py

```diff
@@ -0,0 +1,473 @@
+# (C) Copyright 2021 IBM Corp.
+# (C) Copyright 2021 Inova Development Inc.
+# All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Click Command definition for the listener command group which includes
+cmds to manage indication listeners that are separate invocations of
+pywbemcli representing a listener.
+
+NOTE: Commands are ordered in help display by their order in this file.
+"""
+
+from __future__ import absolute_import, print_function
+
+import subprocess
+import signal
+import argparse
+from time import sleep
+import click
+import psutil
+
+from pywbem import WBEMListener
+
+from .pywbemcli import cli
+from ._common import CMD_OPTS_TXT, GENERAL_OPTS_TXT, SUBCMD_HELP_TXT, \
+    format_table
+from ._common_options import add_options, help_option
+from ._click_extensions import PywbemcliGroup, PywbemcliCommand
+
+DEFAULT_LISTENER_PORT = 5988
+DEFAULT_LISTENER_PROTOCOL = 'http'
+
+LISTEN_OPTIONS = [
+    click.option('--port', type=int, metavar='PORT',
+                 required=False, default=DEFAULT_LISTENER_PORT,
+                 help=u'The port number listened on. '
```

Comment:
How about port number on listener will open to receive indications.

Also note that this can be any available port.  To be honest I never understood whey we set 5988 as defualt

Reply:
DONE - changed help to "The port number the listener will open to receive indications. This can be any available port.". The default port is a discussion item.